Header header
float32 output
float32 output_uncertainty